charts/newrelic-pixie/templates/deployment.yaml

```diff
@@ -58,6 +58,14 @@ spec:
         - name: VERBOSE
           value: "true"
         {{- end }}
+        {{- if (include "newrelic-pixie.lowDataMode" .) }}
+        - name: COLLECT_INTERVAL_SEC
```

Comment:
Does it makes sense to always expose those env vars and add the default to be those if lowDataMode=true?
Or we prefer not to provide that flexibility? No idea if it can create issues

Reply:
These are certainly variables we can expose in the Helm chart if there is demand of that. For now I would keep it simple.